draft-ietf-ppm-dap.md

```diff
@@ -1,5 +1,4 @@
----
-title: "Distributed Aggregation Protocol for Privacy Preserving Measurement"
+--- title: "Distributed Aggregation Protocol for Privacy Preserving Measurement"
```

Comment:
I think this change should be rolled back because it delimits the YAML document for front matter

Reply:
whoops, fat fingers!